htdocs/core/lib/pdf.lib.php

```diff
@@ -1077,7 +1077,7 @@ function pdf_writelinedesc(&$pdf,$object,$i,$outputlangs,$w,$h,$posx,$posy,$hide
  *  @param  int			$issupplierline      Is it a line for a supplier object ?
  *  @return string       				     String with line
  */
-function pdf_getlinedesc($object,$i,$outputlangs,$hideref=0,$hidedesc=0,$issupplierline=0)
+function pdf_getlinedesc($object,$i,$outputlangs,$hideref=0,$hidedesc=0,$issupplierline=0,$showrefcustomerinsupplierline=0)
```

Comment:
What is known into your local system is not the supplier ref. The supplier knows its ref, but yourself, you know your own ref. So you should see first your own ref. You can introduce an option, but can you invert it and name it $hideprodrefifsupplierline (with default value to 0)